cset-workflow/bin/app_env_wrapper

```diff
@@ -7,6 +7,8 @@
 
 set -euo pipefail
 
+echo "app_env_wrapper running in $(hostname)"
```

Comment:
```suggestion
echo "app_env_wrapper running on $(hostname -f)"
```
Print the fully qualified domain name, to prevent it being ambiguous.

Reply:
No, is not site specific. My eyesight is not what it used to be and I missed that line (the one I was looking for). So I will close the ticket as not needed, duplicate, or similar.